src/main/java/org/influxdb/dto/Point.java

```diff
@@ -262,6 +263,9 @@ private void addFieldByAttribute(final Object pojo, final Field field, final Col
         if (column.tag()) {
           this.tags.put(fieldName, (String) fieldValue);
         } else {
+          if ("time".equals(fieldName) && fieldValue instanceof Instant) {
```

Comment:
Hi, i dont want to have hard coded field names in the Point object, you need to find a other way to ensure this is de/serialized in the correct way.